carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -211,6 +139,86 @@ std::shared_ptr<Geofence> WMBroadcaster::geofenceFromMsg(const cav_msgs::Traffic
   return gf_ptr;
 }
 
+void WMBroadcaster::passingControlLineFromMsg(std::shared_ptr<Geofence> gf_ptr, const cav_msgs::TrafficControlMessageV01& msg_v01, const std::vector<lanelet::Lanelet>& affected_llts)
```

Comment:
Lets call this function addPassingControlLineFromMsg since it is not only doing the conversion but also the set operation for the geofence